spec/integration/rating_spec.rb

```diff
@@ -65,32 +65,32 @@
     it 'redirects to the index from the new rating form is cancel is hit', :js => true do
       Capybara.ignore_hidden_elements = false
 
-      visit "/"
+      visit "/projects"
```

Comment:
We will have to change all of these tests. Looking at how many there are, it probably makes sense to do on a different PR.